STAC-extensions.yaml

```diff
@@ -337,7 +337,7 @@ paths:
         - $ref: '#/components/parameters/limit'
         - $ref: '#/components/parameters/ids'
         - $ref: '#/components/parameters/collectionsArray'
-        - $ref: '#/components/parameters/query'
```

Comment:
If this is deprecated rather than removed shouldn't it remain in the OpenAPI doc for now?

Reply:
I wasn't sure. But yeah, happy to leave it in. Is there a way to mark things as deprecated in openapi?

Reply:
Yes, OpenAPI has a deprecated flag you can set in some contexts.

Reply:
Ok, marked as deprecated, added back in.